Feature/Education.feature

```diff
@@ -0,0 +1,20 @@
+﻿Feature: Education
+User wants to add education details into Profile Details
+Acceptance Criteria
+User is able to add the education details
+
+@mytag
+ Scenario: 01 Test Add Education sucessfully
+	Given User enter UniversityName as "Delhi University"
+	And   Select CountryOfUniversity as "India"

```

Comment:
The usage of And keyword is not proper. This is used as a follow up after When keyword.

Furthermore, you used too many And and it's overkill. Imagine adding hundreds of And's for a complex test?

Suggestion: You could just use a When keyword with a Scenario Outline and add "When user enters education details "University Name", "Education Type", "Degree Name"...etc

Please read up on the When keyword.